project_euler/problem_34/sol1.py

```diff
@@ -41,17 +59,17 @@ def sum_of_digit_factorial(n: int) -> int:
 
 def compute() -> int:
     """
-    Returns the sum of all numbers whose
+    Returns the sum of all numbers whose 
     sum of the factorials of all digits
     add up to the number itself.
     >>> compute()
     40730
     """
-    summ = 0
+    the_list = []
     for num in range(3, 7 * factorial(9) + 1):
         if sum_of_digit_factorial(num) == num:
-            summ += num
-    return summ
+            the_list.append(num)

```

Comment:
Could we make this list using a https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions

Reply:
Ok sir. Will do it